src/__tests__/http-test.js

```diff
@@ -1182,7 +1182,7 @@ function urlString(urlParams?: ?{ [param: string]: mixed, ... }) {
           }),
         );
 
-        expect(response.status).to.equal(500);
+        expect(response.status).to.equal(200);
```

Comment:
The library is heavily tested, and this case is the only one affected as intended.